### PR TITLE
Schedule improvements

### PR DESCRIPTION
**Pull request by jomey**
See the commit messages.
Main points:
* Allow Tutorial Leads to be missing in the team.yaml
* Don't show empty descriptions or resources list for schedule entries
* Reformat team.yaml and schedule.yaml according to the `.editorconfig`. Should help to faster identify mistakes.

Also addresses uwhackweek/jupyterbook-template/issues/92

### CODE REVIEW

File: book/schedule.yaml [hunk 1]
```diff
@@ -1,271 +1,199 @@
...
+        leads:
+          - Anthony Arendt
+          - Charley Haley
+          - Jessica Scheick
+      - time: 10:00 - 10:30
+        location: Zoom
+        title: Overview of ICESat-2 Mission
+        description:
+        leads:
+          - TBD
```
Comment (jomey) [inline book/schedule.yaml:19]: Here an example for an entry not in the team.yaml

**Comment by JessicaS11** [comment]
Looks like the newest merges into main have created conflicts with these updates...

**Comment by jomey** [comment]
> Looks like the newest merges into main have created conflicts with these updates...

Resolved

**Review by scottyhq — APPROVED** [review]
thanks @jomey !

**Pull request merged**